core/frontend/src/test/scala/manual/PropertyErrorManualTest.scala

```diff
@@ -206,4 +211,45 @@ object PropertyErrorManualTest {
 //    ImmutableValue.isImmutable[CCB]
 //    ImmutableValue.isImmutable[CCD]
 //  }
+
```

Comment:
Added lines below this one should be deleted - no need to keep commented-out test cases.